src/Tickets/Commerce/Admin_Tables/Orders.php

```diff
@@ -332,6 +334,34 @@ public function column_gateway( $item ) {
 		if ( ! $gateway ) {
 			return $item->gateway;
 		}
+
 		return $gateway::get_label();
 	}
+
+	/**
+	 * Handler for the errors column
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_Post $item
+	 *
+	 * @return string
+	 */
+	public function column_errors( $item ) {
+		$logs = $item->gateway_errors;
+
+		if ( empty( $logs ) ) {
+			return '';
+		}
+
+		if ( ! is_array( $logs ) ) {
+			$logs = unserialize( $logs );
+		}
+
+		$count = count( $logs );
+
+		// @todo figure out how we should present these errors to the site admin.
+		// the errors are in $item->gateway_errors
+		return esc_html__( sprintf( '%d errors', $count ), 'event-tickets' );
+	}
```

Comment:
I would love to see this instead of being a column being an error on the Status.
As a question, what is the relationship between errors and status of the order?

Reply:
I have ideas on how we could do that in a super clean way and fast.

Reply:
There is no direct relation between payment intent errors and the order status. 

If the payment intent is regenerated and the user is allowed to pay with their card, the order statuses will follow the normal flow. 

If we gave up on generating a payment intent two things can happen:

- if the site has PayPal enabled, the user will be able to checkout with PayPal and the order flows normally. Stripe errors will be attached to the PayPal order, meaning that user did not have a choice to use Stripe.
- if the site only has Stripe enabled, the user will be unable to checkout, and thus an order is never created - in this case, we may need to persist these errors in an option.

I lilked the idea of attaching errors to the Orders at first, but I"m reconsidering this, given that the errors are actually generic. In the sense that there's nothing about that potential purchase causing the errors - they are caused by configurations issues, so they will actually happen to all orders.